modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/repo/DetectorRepository.java

```diff
@@ -30,8 +30,28 @@
 @RepositoryRestResource(excerptProjection = InlineType.class)
 public interface DetectorRepository extends PagingAndSortingRepository<Detector, Long> {
 
+    /**
+     * Finds a detector by its unique uuid, if any.
+     *
+     * @param uuid Detector uuid.
+     * @return Detector identified by the unique key.
+     */
     List<Detector> findByUuid(@Param("uuid") String uuid);
 
+    /**
+     * Finds a list of detectors owned by provided owner, if any.
+     *
+     * @param owner Detector owner.
+     * @return List of detectors
+     */
+    List<Detector> findByCreatedBy(@Param("owner") String owner);
```

Comment:
Let's use param "user" instead of "owner", since the createdBy column isn't necessarily the one who owns it. Fine if you want to treat the createdBy user as the owner from the UI, at least for now, but I don't want to conflate those two things in the API itself.